homeassistant/components/ecobee/strings.json

```
@@ -18,7 +18,7 @@
       "token_request_failed": "Error requesting tokens from ecobee; please try again."
     },
     "abort": {
-      "one_instance_only": "This integration currently supports only one ecobee instance."
+      "one_instance_only": "[%key:common::config_flow::abort::single_instance_allowed%]"
```

Please change the key as well and adjust the config flow and its tests accordingly.
```suggestion
      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]"
```